src/guides/v2.3/frontend-dev-guide/layouts/layout-create.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
New topics need to be added to the Table of Contents.

Reply:
@dobooth No idea on how to add this in the Table of Contents. Please guide me on this point.
Thanks.

Reply:
The Table of Contents (TOC) files are here: https://github.com/magento/devdocs/tree/master/src/_data/toc. The one you want is https://github.com/magento/devdocs/blob/master/src/_data/toc/frontend-developer-guide.yml.
From that you can figure out the format and add your new Topic to the correct spot.

Reply:
@dobooth Added TOC for the new page and commit the code changes. Please review it.
Thanks for the help.